hyperspy/learn/mva.py

```diff
@@ -958,6 +959,170 @@ def plot_explained_variance_ratio(self, n=50, log=True):
             ax.semilogy()
         return ax
 
+    def scree_plot(self, n=50, log=True, cutoff=0.01, signal_num=0,
```

Comment:
> as any uninformed guess will always be wrong and it can be misleading to the unaware user

This is a good point. Should we try to provide a good estimate, or just leave that up to the user? I found the following tutorial, which reviews a few different ways to determine the "right" number of components: http://eric.univ-lyon2.fr/~ricco/tanagra/fichiers/en_Tanagra_Nb_Components_PCA.pdf